e2e-tests/custom-login/conf.js

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint import/no-unresolved:0, import/no-extraneous-dependencies:0, no-console:0 */
+/* global jasmine */
+const jasmineReporters = require('jasmine-reporters');
+const daemonUtil = require('../tools/daemon-util');
+
+const promises = Promise.all([
+  daemonUtil.startCustomLoginServer()
+]);
+
+const config = {
+  // Update these parameters to match your environment for tests
+  params: {
+    login: {
+      username: 'george',
+      password: 'Asdf1234',
+      email: 'george@acme.com'
+    }
+  },
+  framework: 'jasmine2',
+  beforeLaunch() {
+    return promises;
+  },
+  onPrepare() {
+    jasmine.getEnv().addReporter(new jasmineReporters.JUnitXmlReporter({
+      savePath: 'build2/reports/junit',
+      filePrefix: 'results',
+    }));
+  },
+  afterLaunch() {
+    promises.then((childProcesses) => {
+      childProcesses.forEach(child => child.stop());
+    });
+    return new Promise(resolve => setTimeout(() => resolve(), 1000));
```

Comment:
Can we make this timeout configurable via an command line argument?

Reply:
+1

Reply:
It's a timeout to wait for app-servers to stop. Not a critical thing, if you ask me. Moreover, I don't think people would pass command line args to protrator cmds as they're invoked by npm scripts. So you will have to update the script to pass it. Not worth the hassle.
I can make it configurable by an env var maybe.

Reply:
Made it configurable through env var